### PR TITLE
Basic about us section added

### PR DESCRIPTION
**Pull request by vjs3**
About us has been added using alert-dialogue

### CODE REVIEW

**Comment by vjs3** [comment]
@creativepsyco on creating the new branch and resetting to desired head this new pull request created but with correct output, it would be great if we can go ahead with this so that for new work I can do things properly.But if this is not the correct way then please let me know i will redo the process.
#150 #170

File: app/build.gradle [hunk 1]
```diff
@@ -76,7 +76,6 @@ dependencies {
     compile "com.android.support:design:${SUPPORT_LIB_VERSION}"
     compile 'com.squareup.picasso:picasso:2.5.2'
     compile 'com.squareup:otto:1.3.8'
-
     googleplayCompile 'com.google.android.gms:play-services-maps:8.3.0'
```
Comment (creativepsyco) [inline app/build.gradle]: empty line

**Comment by creativepsyco** [comment]
You branch has conflicts. Please do `git fetch parent && git rebase parent/master` where `parent` is the current fork. Resolve and force push.

File: app/src/main/java/org/fossasia/openevent/activities/MainActivity.java [hunk 2]
```diff
@@ -281,6 +287,26 @@ public void run() {
                     }
                 });
                 mDrawerLayout.closeDrawers();
+
+                startActivity(intent);
```
Comment (creativepsyco) [inline app/src/main/java/org/fossasia/openevent/activities/MainActivity.java]: why is this line here? Is it part of the patch?

**Comment by creativepsyco** [comment]
Your code has several empty lines, please fix. Check the diff to get an idea.

**Comment by vjs3** [comment]
@creativepsyco I have did the force push please review now.

**Comment by vjs3** [comment]
@creativepsyco you must be busy but please let me know do I have to resolve merge conflicts again? as yesterday when I submitted it then it was not showing any merge conflict.I am stuck with this please let me get over with it so that I can solve other bugs as well :smile: .
Thanks!

**Comment by creativepsyco** [comment]
See PR #186

**Comment by creativepsyco** [comment]
Merged via PR #186
